src/components/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import RestaurantList from './RestaurntList';
+import './App.css';
+
+const App = () => {
+    return (
+        <div className="ui container grid app">
```

Comment:
Consider using [classnames](https://github.com/JedWatson/classnames).

Reply:
Got the main idea, but I don't think it's useful here